scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/RestResponseController.java

```diff
@@ -102,11 +115,14 @@
    * @param request
    * @return
    */
-  @PostMapping(
-      value = "/runs/progressive",
-      produces = {"application/json"})
+  @PostMapping("/runs/progressive")
+  @ApiOperation(
+      value = "Progressive scan of runs",
+      response = ca.on.oicr.gsi.runscanner.dto.ProgressiveResponseDto.class)
   @ResponseBody
-  public ProgressiveResponseDto progressive(@RequestBody ProgressiveRequestDto request) {
+  public ProgressiveResponseDto progressive(
+      @ApiParam(value = "ProgressiveRequestDto object created by requesting client") @RequestBody
```

Comment:
"Object containing request options" might be a better description? All parameters are, by definition, created/provided by the client